Album/ViewControllers/ProfileControllers/ProfileViewController.m

```diff
@@ -35,31 +38,56 @@ - (void)viewDidLoad {
     [super viewDidLoad];
     // Set API helper
     self.apiHelper = [[ParseAPIHelper alloc] init];
+    self.colorConvertHelper = [[ColorConvertHelper alloc] init];
```

Comment:
We do not this anymore because now the helpers are static functions

Reply:
Oops. Done!